test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -26,7 +26,9 @@ using namespace solidity::langutil;
 using namespace solidity::frontend;
 using namespace solidity::frontend::test;
 
-SMTCheckerTest::SMTCheckerTest(std::string const& _filename): SyntaxTest(_filename, EVMVersion{})
+SMTCheckerTest::SMTCheckerTest(std::string const& _filename):
+	SyntaxTest(_filename, EVMVersion{}),
```

Comment:
I see it was already here, but shouldn't we be using the EVM version specified on the command line? I.e. the one you can get from `solidity::test::CommonOptions::get()`.

Reply:
I don't know :)

Seems like it should be as you suggest :)

Reply:
It depends on what the EVM version is used here for. Maybe it just does not matter for SMTChecker. But then it also does not hurt to pass in the right one.

We do have some cases where we explicitly ignore it - e.g. gas tests because we want expectation to match the latest version (we can't test all of them) - but I think that it's best to respect the CLI flags (`--evm-version`, `--optimize`, `--abiencoderv1`, `--eof-version`, etc.) unless there's a good reason not to.

Reply:
```suggestion
	SyntaxTest(_filename, CommonOptions::get().evmVersion()),
```